db/schema_tables.cc

```diff
@@ -104,6 +104,7 @@ namespace db {
 
 schema_ctxt::schema_ctxt(const db::config& cfg)
```

Comment:
I'm not reviewing any commits that don't have explanations any more.

Reply:
I forgot - sorry about that.

Reply:
The explanation repeats the subject line and adds no information.

Reply:
What else would you like to see here? I've already explained why in 2-3 previous patches. Do you want me to repeat the same explanation in every commit?

Reply:
Of course mindless repetition isn't helpful. But this patch does more than just add a field to the schema, it also handles rolling upgrades in a non-trivial way. Suppose that breaks and gets bisected to this patch, the lucky debugger will get zero help from the changelog about how this is supposed to work.

Reply:
I gave it another try. Hopefully, now it's more comprehensive. @avikivity what do you think?